region/maridia/inner-pink/Draygon's Room.json

```
@@ -302,10 +302,15 @@
       "name": "Microwave Fight",
       "requires": [
         "h_canNavigateUnderwater",
-        "h_canBreakThreeDraygonTurrets",
+        {"or": [
+          "canTrickyDodgeEnemies",
+          "h_canBreakThreeDraygonTurrets"
```

```suggestion
        {"or": [
          {"and": [
            "canTrickyDodgeEnemies",
            {"or": [
              "Gravity",
              "canInsaneJump",
              {"enemyDamage": {"enemy": "Draygon", "type": "turretProjectile", "hits": 1}}
            ]}
          ]},
          "h_canBreakThreeDraygonTurrets"
```
- You can't react dodge the turrets, so you need to know what you are doing.  Waiting on the left side of the room works well
- Dying in 1 hit is very uncomfortable.  Being able to survive a hit helps a lot.
- Turrets can RNG snipe you right when draygon swoops and there is nothing you can do.